cli/bin/sg

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
```

Comment:
This requires a changelog entry telling people semgrep is now also accessible as the shorter
'sg'
Maybe @milanwilliams has an opinion about this (I am personally in favor).

Reply:
I'm open to adding a changelog entry in this PR, but I figured we could defer before we make an official announcement (e.g. as noted in the public-facing docs).

Earlier I had set an alias via my `.zshrc` config
```sh
alias sg="/opt/homebrew/bin/semgrep"
```
and this new file is functionally equivalent (w/ one fewer step).